Scripts/Data/Operation/TransformDirectionExtractor.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace Zinnia.Data.Operation
+{
+    using UnityEngine;
+
+    /// <summary>
+    /// Extracts the chosen direction.
+    /// </summary>
+    public class TransformDirectionExtractor : TransformVector3PropertyExtractor
+    {
+        /// <summary>
+        /// The direction axes of the transform.
+        /// </summary>
+        public enum AxisDirection
+        {
+            /// <summary>
+            /// The axis moving right from the transform origin.
+            /// </summary>
+            right,
+            /// <summary>
+            /// The axis moving up from the transform origin.
+            /// </summary>
+            up,
+            /// <summary>
+            /// The axis moving forward from the transform origin.
+            /// </summary>
+            forward
+        }
+
+        [Tooltip("The direction to extract from the Transform."), SerializeField]
+        private AxisDirection _extractDirection;
+        /// <summary>
+        /// The direction to extract from the Transform.
+        /// </summary>
+        public AxisDirection ExtractDirection
```

Comment:
Naming convention: Nouns for data members, not verbs. `ExtractionDirection`? Just `Direction` also works as the script name already implicitly namespaces it into `Extraction`.

Also update the backing field's name.